src/selectors/resourceSelectors.js

```diff
@@ -1,3 +1,40 @@
 export const rootResource = state => Object.values(state.selectorReducer.resource)[0]
```

Comment:
This module needs a copyright statement.

Reply:
created #775.

is it ok to merge this one in light of that?

Reply:
Sure, resolving now.